### ISSUE TITLE
[Feature] Support for LZMA vtk files

### ISSUE THREAD

**Issue by skoloCFD** [Kitware/vtk-js#3042]
### Feature Request

Given the smaller filesize of vtk encoded with LZMA, and the support in paraview/vtk-python for this compression type, native support in vtk.js could be useful.

### Motivation and Detailed Description

See above

**Comment by daker** [comment]
Thanks for the report, maybe if you could attach an example file in case someone wants to work on that.

**Comment by skoloCFD** [comment]
Certainly! Here's a small example - zipped because github won't accept .vtp uploads.

[vtp_LZMA_example.vtp.zip](https://github.com/Kitware/vtk-js/files/14742081/vtp_LZMA_example.vtp.zip)

**Comment by daker** [comment]
@jourdain what do you recommend to use ? i was thinking about using [Lzip](https://www.nongnu.org/lzip/) but this means creating a package and publish it on npm
